src/Microsoft.AspNet.SignalR.Redis/RedisConnection.cs

```diff
@@ -88,8 +88,11 @@ public Task ScriptEvaluateAsync(int database, string script, string key, byte[]
                 }
 
                 var redisResult = await _connection.GetDatabase(database).ScriptEvaluateAsync(
-                   @"local newvalue = redis.call('GET', KEYS[1])
-                    if newvalue < ARGV[1] then
```

Comment:
Why not change only the if condition to something like: 

```
if not newvalue or newvalue < ARGV[1] then
```
additional explicit check?